projects/1.16/assets/wandering-bag/wandering_bag/lang/zh_cn.json

```
@@ -0,0 +1,29 @@
+{
+  "itemGroup.tabwandering_bag_tab": "流浪商人包",
+  "item.wandering_bag.tamed_ravager_spawn_egg": "驯服的劫掠兽刷怪蛋",
+  "entity.wandering_bag.tamed_ravager": "驯服的掠夺兽",
+  "item.wandering_bag.ingot_from_other_world": "§e异世之锭",
+  "item.wandering_bag.hammer_from_farlands": "§e边境之锤",
+  "item.wandering_bag.herb_of_rescue": "§b救援药草",
+  "item.wandering_bag.flask_of_adaptation": "§d适应之瓶",
+  "item.wandering_bag.dagger_of_traitor": "§b背叛之刃",
+  "item.wandering_bag.wither_bone_meal": "§d凋零骨粉",
+  "item.wandering_bag.spell_pr": "Spell Pr",
+  "item.wandering_bag.bad_tome": "§b未定符咒",
+  "item.wandering_bag.whip_of_taming": "§b驯服之鞭",
+  "item.wandering_bag.homemade_totem": "§e自制图腾",
+  "item.wandering_bag.pillager_bag": "§b掠夺者包",
+  "item.wandering_bag.bucket_of_unknown": "§e未知之桶",
+  "item.wandering_bag.spices_from_elsewhere": "§e异地香料",
+  "item.wandering_bag.mirror_of_abundance": "§d丰盈之镜",
+  "item.wandering_bag.uncasted_spell": "§d未定符咒",
+  "item.wandering_bag.strange_mixture": "§d怪异药水",
+  "item.wandering_bag.wandering_bag_loot": "§e流浪商人包",
+  "item.wandering_bag.witch_bag": "§d女巫包",
+  "subtitles.ravager_horn": "掠夺号角",
```

```suggestion
  "subtitles.ravager_horn": "劫掠兽角：吹奏",
```
是不是和[山羊角](https://minecraft.fandom.com/zh/wiki/%E5%B1%B1%E7%BE%8A%E8%A7%92)类似的吹奏声音字幕？

---

是的，感谢指教